### PR TITLE
Adding possibilty to study D0 reflections and D0->KK signal for D* tag & probe studies.

### PR DESCRIPTION
**Pull request by arossi81**

### CODE REVIEW

**Comment by arossi81** [comment]
This commit requires approval by @fgrosa

**Review by mfaggin — APPROVED** [review]
Hi @arossi81 thanks for the development. Everything looks clear to me and I do not have anything major to comment, I approve it. However, let's wait for @fgrosa since you asked for his review.

Just one question: here you want to study the reflections in a "data driven" way, namely you flag a candidate as reflected simply checking if the daughter PID is compatible with the reflected hypotheses, w/o any check to MC truth, right?

**Review by fgrosa — APPROVED** [review]
It looks very good to me, thanks @arossi81!

**Pull request merged**
